opentech/apply/dashboard/templates/dashboard/reviewer_dashboard.html

```diff
@@ -36,6 +36,39 @@ <h1 class="gamma heading heading--no-margin heading--bold">Dashboard</h1>
 
         {% endif %}
     </div>
+
```

Comment:
Maybe add a "if my_submissions" check to only show this if the reviewer has submissions, I believe most will not have any.

Reply:
Thanks, updated.